charts/karpenter/values.yaml

```
@@ -156,24 +165,30 @@ settings:
   # faster than this time, the batching window will be extended up to the maxDuration. If they arrive slower, the pods
   # will be batched separately.
   batchIdleDuration: 1s
+  # -- How the Karpenter scheduler should treat preferences. Preferences include preferredDuringSchedulingIgnoreDuringExecution
+  # node and pod affinities/anti-affinities and ScheduleAnyways topologySpreadConstraints. Can be one of 'Ignore' and 'Respect'
+  preferencePolicy: Respect
   # -- Cluster CA bundle for TLS configuration of provisioned nodes. If not set, this is taken from the controller's TLS configuration for the API server.
   clusterCABundle: ""
   # -- Cluster name.
   clusterName: ""
   # -- Cluster endpoint.
   clusterEndpoint: ""
-  # -- The VM memory overhead as a percent that will be subtracted from the total memory for all instance types
+  # -- The VM memory overhead as a percent that will be subtracted from the total memory for all instance types. The value of `0.075` equals to 7.5%.
```

hyper-hit (matching other quotes):
```suggestion
  # -- The VM memory overhead as a percent that will be subtracted from the total memory for all instance types. The value of '0.075' equals to 7.5%.
```